main.cpp

```diff
@@ -287,7 +287,7 @@ int main(void)
         } else if (in_char == 'r') {
             (void) fcc_storage_delete(); // When 'r' is pressed, erase storage and reboot the board.
             printf("Storage erased, rebooting the device.\n\n");
-            ThisThread::sleep_for(1*1000);
+            ThisThread::sleep_for(1s);
```

Comment:
ThisThread::sleep_for(1s); I think this should be ThisThread::sleep_for(1);

Reply:
mbed-os warning suggested using `1s`

Reply:
It's the new Chrono API in Mbed OS: https://en.cppreference.com/w/cpp/chrono/operator%22%22s